shotover-proxy/src/transforms/util/unordered_cluster_connection_pool.rs

```diff
@@ -113,10 +113,10 @@ async fn rx_process<C: CodecReadHalf>(
     codec: C,
 ) -> Result<()> {
     let mut in_r = FramedRead::new(read, codec);
-    let mut return_channel_map: HashMap<u16, (tokio::sync::oneshot::Sender<Response>, Message)> =
+    let mut return_channel_map: HashMap<i16, (tokio::sync::oneshot::Sender<Response>, Message)> =
         HashMap::new();
 
-    let mut return_message_map: HashMap<u16, Message> = HashMap::new();
+    let mut return_message_map: HashMap<i16, Message> = HashMap::new();
 
     // let foo = timeout(Duration::from_millis(10), )
```

Comment:
Might as well clean this up in this PR

Reply:
Also `// info!("tjos happened");` in the same file.

Reply:
I put that cleanup into https://github.com/shotover/shotover-proxy/pull/387